sound/smooth_swing_v2.h

```diff
@@ -26,7 +26,16 @@ class SmoothSwingV2 : public SaberBasePassThrough {
       accent_swings_present = true;
     } else {
       accent_swings_present = false;
-      STDOUT.print("Accent Swings NOT Detected: ");
+      STDOUT.println("Accent Swings NOT Detected: ");
+    }
+    if (slsh.files_found() > 0 && smooth_swing_config.AccentSlashAccelerationThreshold > 0.0) {
```

Comment:
Accent slashes don't make any sense without accent swings, right?
If so, I think this entire if statement should be moved up to inside the "Accent swings detected" code block.